src/MOI/indicator_constraint.jl

```diff
@@ -0,0 +1,117 @@
+
+function MOI.supports_constraint(
+    ::Optimizer, ::Type{MOI.VectorAffineFunction{Float64}}, ::Type{<:MOI.IndicatorSet{A, S}}
+) where {A, S <: Union{MOI.LessThan{Float64}, MOI.GreaterThan{Float64}, MOI.EqualTo{Float64}}}
+    return true
+end
+
+function MOI.is_valid(
+    model::Optimizer,
+    c::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}, S}
+) where {A, S <: MOI.IndicatorSet}
+    info_tuple = get(model.indicator_constraint_info, c.value, nothing)
+    if info_tuple === nothing
+        return false
+    end
+    return typeof(info_tuple[1].set) == S
+end
+
+function MOI.get(model::Optimizer,
+                 ::MOI.ConstraintSet,
+                 c::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}, MOI.IndicatorSet{A, S}}
+) where {A, S}
```

Comment:
Sorry to be a pain, but can we have consistent formatting? Also, we prefer to drop type parameters if not needed in the function.
```Julia
function MOI.get(
    model::Optimizer,
    ::MOI.ConstraintSet,
    c::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}, <:MOI.IndicatorSet}
)
```

Reply:
what formatting do you refer to, the parenthesis on a new line?

Reply:
Dropping arguments to a new line and indenting 4 spaces, with parenthesis on new line.

Reply:
addressed in last commit